### PR TITLE
[apps] Adds migrations of the apps' contracts to Ropsten

### PR DESCRIPTION
**Pull request by cf19drofxots**
### Description

This adds the migration data (primarily deployed contract addresses) on Ropsten.

It also adds the tooling used to do said migration.

### CODE REVIEW

**Comment by cf19drofxots** [comment]
@snario we _could_ but what's the added benefit for our use case?

**Comment by snario** [comment]
> @snario we _could_ but what's the added benefit for our use case?

Based on `ethers` so the code is cleaner; has all the same functionality and features; much smaller dependency

**Comment by cf19drofxots** [comment]
I spent a little bit of time to see if refactoring to use etherlime is super trivial.

It's not quite and it'll take a bit more time, so will come back around to it later - not a priority atm.

File: packages/apps/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+| ---------- | --------------- |
+| 1          | Main net        |
+| 3          | Ropsten testnet |
+| 4          | Rinkeby testnet |
+| 42         | Kovan testnet   |
+
+Not all of the networks will be used for the Counterfactual contracts, but you can find a more comprehensive list [here](https://ethereum.stackexchange.com/a/17101).
+To run a migration against a target network:
+
+- make sure the target network configuration exists in `truffle.js`
```
Comment (Alonski) [inline packages/apps/README.md]: Should be `truffle-config.js`?

Comment (cf19drofxots) [reply packages/apps/README.md]: It should be both! good catch

**Comment by cf19drofxots** [comment]
Have to rerun migration since https://github.com/counterfactual/monorepo/pull/584 landed before this PR

**Pull request merged**